privacy/i2p.md

```diff
@@ -0,0 +1,70 @@
+Using I2P (Invisible Internet Project) with Qubes
+=================================================
+
+"The I2P network provides strong privacy protections for communication over the Internet. Many activities that would risk your privacy on the public Internet can be conducted anonymously inside I2P." - [I2P Website](https://geti2p.net/)
+
+
+Preparation
+===========
```

Comment:
(This point got lost in initial review submission.)
It would be best to describe this as "Preparation - Basic" that is similar to instructions on the i2p site. This means the user will not get some of the added protection that is possible with Qubes isolation. In particular, it is risky to run Firefox (HTML5 + Javascript document interpreter) in same VM as i2p-router. Even when you consider that i2p-router has some default internal apps like bittorrent, those are still much less risky than a web browser, IMHO.

IOW, this setup is not to i2p what Whonix is to Tor.

Options are to add an "Advanced Notes" section to the bottom describing in general terms what it takes to setup i2p-router in a proxyVM and Firefox config in appVM(s); also could describe this type of setup in detail. @adrelanos input may be helpful here.